tests/Integration/RequestIntegrationTest.php

```diff
@@ -31,20 +36,29 @@ public function testConstructorThrowsDeprecationIfPassingOptions(): void
      * @group legacy
```

Comment:
This method should not use anything deprecated, so this line should be unnecessary